galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -453,3 +453,12 @@ messages:
     level: Warning
     description: "Application pods should not run as user ID (UID) 1337"
     template: "User ID (UID) 1337 is reserved for the sidecar proxy."
+
+  - name: "ConflictingGateways"
+    code: IST0145
+    level: Error
+    description: "Gateway should not have the same selector, port number and matched hosts of server"
+    template: "Conflicting gateways are detected for gateways %s on selector, port number and hosts."
```

Comment:
It will be good to specify selector, port number, and host in the description?

```
Error [IST0145] (Gateway alpha.default) Conflicting gateways are detected for gateway default/alpha to default/gama,default/theta,default/beta
on selector 'ingressgateway' with port number '80' and hosts 'foo.bar'.
```

Reply:
okay, that's great to make this error information more useful